api/swagger/swagger.yaml

```diff
@@ -551,6 +551,11 @@ definitions:
         type: string
         description: Date when a certification of exemption has been filed for the trial
         format: date-time
+      study_phase:
+        type: array
+        description: Phases of the study
+        items:
+          type: string
```

Comment:
Yeah, that's because I wasn't sure at the moment, with all the variations :sweat_smile:
I'll add 1-2 samples now, that I've read the code for the phase normalisation PR :+1: